legal-api/src/legal_api/reports/report.py

```diff
@@ -295,6 +295,10 @@ def _handle_special_resolution_filing_data(self, filing):
             self._format_special_resolution_application(filing, "alteration")
 
     def _set_completing_party(self, filing):
+        business = BusinessService.fetch_business_by_filing(self._filing)
+        if business.is_alternate_name_entity and (business.entity_type == LegalEntity.EntityTypes.SOLE_PROP.value):
+            return
+
```

Comment:
I believe a SP still has a completing party

Reply:
I think we return the party info for the only `proprietor` role.

https://github.com/kzdev420/lear/blob/1d171cb28fc5044cbe40014ea89a3e8eb20a4266/legal-api/src/legal_api/models/alternate_name.py#L206-L219

Reply:
No, this is specifically for the completing party.  The proprietor party info is different.  Also, the proprietor is not necessarily the same as the completing party if that's what you were thinking.

I think it may work as is if you just remove the new lines of code you added.  

![image](https://github.com/bcgov/lear/assets/6685223/ea8ddc0e-96e7-4ed7-9714-e8d48d63d94e)

Reply:
Okay